src/ansys/geometry/core/math/vector.py

```diff
@@ -265,6 +266,21 @@ class representing the second point.
         """
         return Vector3D(point_b - point_a)
 
+    @staticmethod
+    def rotate_vector(_this: "Vector3D", vector: "Vector3D", angle: float) -> "Vector3D":
```

Comment:
Do not call it ``_this``. That doesn't make sense. A static method is not related to any instance. So there is no concept of ``this``, ``self`` or anything like that. Rename to ``vec_a`` and ``vec_b`` for example.

Reply:
taking this out of scope of this PR